src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -650,6 +650,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5037 = 5037,
 
+        /// <summary>
+        /// Waring_IconURLDeprecationMessage
```

Comment:
```suggestion
        /// Warning_IconURLDeprecationMessage
```

Reply:
I will change to its respective string resource.